include/openenclave/internal/eeid.h

```diff
@@ -136,11 +136,7 @@ oe_result_t oe_remeasure_memory_pages(
  *
  */
 oe_result_t verify_eeid(
-    const uint8_t* reported_enclave_hash,
-    const uint8_t* reported_enclave_signer,
-    uint16_t reported_product_id,
-    uint32_t reported_security_version,
-    uint64_t reported_attributes,
+    const oe_eeid_relevant_sgx_claims_t* relevant_claims,
```

Comment:
verify_eeid is a non-sgx specific name so if you use an sgx specific type, the function should be renamed in line 138 to put sgx in the name.  (And it should be in an sgx specific header file)